src/hdnode.js

```diff
@@ -291,6 +291,30 @@ HDNode.prototype.isNeutered = function () {
   return !(this.keyPair.d)
 }
 
+HDNode.prototype.derivePath = function (path) {
+  typeforce(types.Bip32Path, path)
+
+  var splitPath = path.split('/')
+  if (splitPath[0] === 'm') {
+    if (this.parentFingerprint) {
+      throw new Error('Not a master node')
```

Comment:
Note,  I might change this error message in near future versions.  Consider unstable